domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/retur/KunneIkkeReturnereSøknadsbehandling.kt

```diff
@@ -0,0 +1,16 @@
+package no.nav.su.se.bakover.domain.søknadsbehandling.retur
+
+import no.nav.su.se.bakover.domain.søknadsbehandling.Søknadsbehandling
+import kotlin.reflect.KClass
+
+sealed interface KunneIkkeReturnereSøknadsbehandling {
+    data object FantIkkeBehandling : KunneIkkeReturnereSøknadsbehandling
+    data object KunneIkkeOppretteOppgave : KunneIkkeReturnereSøknadsbehandling
+    data object FantIkkeAktørId : KunneIkkeReturnereSøknadsbehandling
+
+    data class UgyldigTilstand(
+        val fra: KClass<out Søknadsbehandling>,
```

Comment:
Ser ikke at vi har behov for parameteret her? kan feks da gjøre det om til `class UgyldigTilstand(): KunneIkkeReturnereSøknadsbehandling`